lib/src/main/java/graphql/nadel/util/MapUtil.kt

```diff
@@ -0,0 +1,38 @@
+package graphql.nadel.util
+
+internal object MapUtil {
```

Comment:
Is this used? in IDE I can't find references

Reply:
Ha, left over from some experiment I was doing.